weblab/.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
```

Comment:
What service do we need to enable to see the results of this in GitHub?

Reply:
http://codecov.io/ ?

Reply:
I've started this off, but it needs some config in the `.travis.yml` too; will get to that next week probably!